semstr/conversion/ptb.py

```diff
@@ -0,0 +1,282 @@
+from ucca import core, layer0, layer1
+from ucca.layer1 import EdgeTags
+from .format import FormatConverter
+
+import sys
+import re
+
+
+class PtbConverter(FormatConverter):
+    def __init__(self):
+        self.node_by_id = {}
+        self.terminals = []
+        self.pending_nodes = []
+        self.node_ids_with_children = set()
+        self.sentence_id = 1
+        self.passage_id = 3
+
+    def _build_passage(self, stream):
+        # p = core.Passage(self.sentence_id or self.passage_id)
+        p = core.Passage(self.passage_id)
+        l0 = layer0.Layer0(p)
+        l1 = layer1.Layer1(p)
+        paragraph = 1
+
+        next(self.parse(stream))
+
+        # add normal nodes
+        self.pending_nodes = list(reversed(self.pending_nodes))
+        while self.pending_nodes:
+            for i in reversed(range(len(self.pending_nodes))):
+                parent_id, edge_tag, node_id = self.pending_nodes[i]
+                parent = self.node_by_id.get(parent_id, -1)
```

Comment:
I think it's better to use `None` instead of `-1` here. Note that `None` is the default in the dict `get` method, so you can just call `self.node_by_id.get(parent_id)` without a second argument to `get`.